src/services/CraftAssetsService.php

```diff
@@ -116,4 +132,23 @@ public function getAssetsByAltTextFilter(bool $hasAltText, array $assetIds = [])
         return $resultAssets;
     }
 
+    private function getVolumesDisabledIds()
+    {
+        $disabledIds = [];
+        $settings = AltTextLab::getInstance()->getSettings();
+
+        $disabledUids = $settings->disabledVolumeUids ?? [];
+        if (!empty($disabledUids)) {
+            $volService = Craft::$app->getVolumes();
+
+            foreach ($disabledUids as $uid) {
+                if ($volume = $volService->getVolumeByUid($uid)) {
+                    $disabledIds[] = $volume->id;
+                }
+            }
+        }
```

Comment:
For better readability, please rename the variables so their purpose is explicit. For example:

`$disabledUids` -> `$disabledVolumeUid`

`foreach ($disabledUids as $uid)` -> `foreach ($disabledVolumeUids as $disabledVolumeUid)`

Please apply the same naming throughout the PR: